frontend/src/app/[locale]/events/[eventSlug]/surveys/[surveySlug]/responses/summary/SelectFieldSummaryComponent.tsx

```diff
@@ -0,0 +1,83 @@
+import { Choice, SelectFieldSummary } from "@/components/SchemaForm/models";
+import { Translations } from "@/translations/en";
+
+interface SelectFieldSummaryComponentProps {
+  translations: Translations;
+  choices: Choice[];
+  fieldSummary: SelectFieldSummary;
+  showMissingResponses?: boolean;
+}
+
+export default function SelectFieldSummaryComponent({
+  translations,
+  choices,
+  fieldSummary,
+  showMissingResponses,
+}: SelectFieldSummaryComponentProps) {
+  const { countResponses, countMissingResponses, summary } = fieldSummary;
+  showMissingResponses ??= true;
```

Comment:
This can be done in the destructuring, can't it? `showMissingResponses = true,`?

Reply:
not in my childhood

syntax these days

grumble grumble

Reply:
siä oli menny lapsi pesuvedem mukana, kaivoiv viemäristä